### DISCUSSION TITLE
Duda sobre listas en Java

### DISCUSSION THREAD

**Discussion by LeoCubelli** [dds-jv/foro#237] [Consultas de objetos] [answered]
Buenas, tenemos una duda con la 2da entrega del TPA, tenemos una lista de usuarios del tipo Usuario, y las clases Duenio y UsuarioAdmin que heredan de Usuario, hay alguna forma de filtrar la lista padre del tipo Usuario a una lista del tipo Duenio para usar sus métodos? Probamos casteando la lista padre y haciendo un filter de instanceof pero no salió.

**Comment by gastonprieto** [comment, marked as answer]
Buenas @LeoCubelli 

Por lo que decís eso huele a un type test, lo cual muy probablemente sea algo a refactorizar. Revisa el modelo con tu ayudante, porque tal vez es una jerarquía de herencia que no es adecuada.
